packages/react-internal/src/components/SearchBox/SearchBox.test.tsx

```diff
@@ -177,4 +177,38 @@ describe('SearchBox', () => {
         .getAttribute('value'),
     ).toBe('0');
   });
+
+  it('invokes escape and clear callbacks on escape keydown', () => {
+    const onEscape = jest.fn();
+    const onClear = jest.fn();
+    const keyDownEvent = new CustomEvent('keydown');
+    (keyDownEvent as any).which = KeyCodes.escape;
+
+    wrapper = mount(<SearchBox onEscape={onEscape} onClear={onClear} />);
```

Comment:
With another change that @behowell has in progress, you'll need to provide a default value for this test to work.
```suggestion
    wrapper = mount(<SearchBox defaultValue="foo" onEscape={onEscape} onClear={onClear} />);
```

Reply:
With my change, onEscape will be called with or without a value, but onClear will only be called if there's a value. 

@richardkooiman feel free to leave this as you have it here, and I'll update it along with my change.

Reply:
@behowell Do you mean you can and want to process both the suggested changes (defaultValue and onEscape) mentioned? I'd appreciate that. (And would it be possible to change the expect mock calls to the built-in matcher aswell? I copied this from the tests already present in the same file and wasn't aware of these methods. However if you prefer I can of course change it myself.

/cc @ecraig12345

Reply:
Sure, I can make both changes when I merge your change with my PR.

Reply:
Thanks!

@ecraig12345 Would that be okay with you? @behowell will then process your comments.